? .test
? Makefile
? Makefile.in
? aclocal.m4
? autom4te.cache
? config.log
? config.status
? configure
? globus.version
? libtool
? doc/Makefile
? doc/apidoc/Makefile
? doc/apidoc/Makefile.in
? doc/man/Makefile
? interface/Makefile
? project/compile
? project/depcomp
? project/install-sh
? project/missing
? project/module-version.m4
? src/Makefile
? src/c/.deps
? src/c/.libs
? src/c/Makefile
? src/c/delegation_simple_api.lo
? src/c/libglite_data_delegation_api_simple_c.la
? src/tools/.deps
? src/tools/.libs
? src/tools/Makefile
Index: bootstrap
===================================================================
RCS file: /cvs/glite/org.glite.data.delegation-cli/bootstrap,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 bootstrap
--- bootstrap	7 Oct 2005 12:23:28 -0000	1.1.1.1
+++ bootstrap	5 Jul 2011 17:05:49 -0000
@@ -7,7 +7,7 @@
 
 mkdir -p src/autogen
 set -x
-aclocal -I project
+aclocal -I project -I /usr/share/glite-build-common-cpp/m4 -I /usr/share/glite-data-build
 libtoolize --force
 autoheader
 automake --foreign --add-missing --copy
